File: MvvmRecycleview/src/main/java/com/ibot/cyranosystems/mvvmrecycleview/model/News.java
package com.ibot.cyranosystems.mvvmrecycleview.model;

/**
 * Created by innobot-linux-7 on 12/11/19,12,cleanArchitectureWitMvvm.
 */
public class News {
    public String Title,Desc;

    public News(String title, String desc) {
        Title=title;
        Desc=desc;
    }

    public News() {
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title=title;
    }

    public String getDesc() {
        return Desc;
    }

    public void setDesc(String desc) {
        Desc=desc;
    }
}

File: KotlinFirstApp/src/main/java/com/ibot/cyranosystems/kotlinfirstapp/model/News.kt
package com.ibot.cyranosystems.kotlinfirstapp.model

/**
 * Created by innobot-linux-7 on 13/11/19,13,cleanArchitectureWitMvvm.
 */
class News {
    var title:String?=null
    var desc:String?=null

    constructor(title: String?, desc: String?) {
        this.title = title
        this.desc = desc
    }
}